docs/topics/installing-cli-tool.adoc

```diff
@@ -20,7 +20,22 @@ However, if you want to analyze applications in languages other than Java or, fo
 
 .Prerequisites
 
-* Red Hat Container Registry Authentication for `registry.redhat.io`. Red Hat distributes container images from `registry.redhat.io`, which requires authentication. For more details, see link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication].
+The following are the prerequisites for the {ProductFullName} CLI installation:
+
+* Java Development Kit (JDK) is installed. {ProductShortName} supports the following JDKs:
+
+** OpenJDK 17 or later
+** Oracle JDK 17 or later
+** Eclipse Temurin™ JDK 17 or later
+
+* 8 GB RAM is required for the {ProductShortName} {CLIName}.
+
+
+Linux installation requires a full Java JDK installed, as having only the Java Runtime (JRE) installed causes issues during analysis.
+
+* To install the {ProductShortName} {CLIName}, you must have a valid username and password for Red Hat Container Registry Authentication, `registry.redhat.io`. 
++
+Red Hat distributes container images from `registry.redhat.io`, which requires authentication. For more details, see link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication].
```

Comment:
This is required if user wants to use a container runtime, which is required if user wants to run java analysis in legacy mode or any other analysis. It is not required for running in **local** mode

Reply:
One more thing, running in **local** mode alse requires Mavan and $JAVA_HOME configured properly, relevant for EACH platform